storio-content-resolver/src/androidTest/java/com/pushtorefresh/storio/content_resolver/impl/DeleteTest.java

```diff
@@ -0,0 +1,52 @@
+package com.pushtorefresh.storio.content_resolver.impl;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import com.pushtorefresh.storio.contentresolver.operation.delete.DeleteResults;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(AndroidJUnit4.class)
+public class DeleteTest extends BaseTest {
+
+    @Test public void deleteOne() {
+        final User user = putUser();
+        oneUserInStorageCheck(user);
+        deleteUser(user);
+        noUsersInStorageCheck();
+    }
+
+    @Test public void deleteCollection() {
+        final List<User> allUsers = putUsers(10);
+
+        final List<User> usersToDelete = new ArrayList<User>();
+        for (int i = 0; i < allUsers.size(); i += 2) {  // I will delete every second

```

Comment:
I think it's not good to write comments like "I ...", it's source code that will be read by many humans, so it's better to write comments without personalisation :)

Fix variant: "Delete every second user"